lib/tsc.js

```diff
@@ -47222,7 +47222,7 @@ var ts;
     var asyncValues = {
         name: "typescript:asyncValues",
         scoped: false,
-        text: "\n            var __asyncValues = (this && this.__asyncIterator) || function (o) {\n                if (!Symbol.asyncIterator) throw new TypeError(\"Symbol.asyncIterator is not defined.\");\n                var m = o[Symbol.asyncIterator];\n                return m ? m.call(o) : typeof __values === \"function\" ? __values(o) : o[Symbol.iterator]();\n            };\n        "
```

Comment:
do not change files in `lib\*` these are generated as part of the build. please remove these from the PR.

Reply:
Removed all changes to the generated files in `lib\*`. Thanks for the hint.